docs/components/FlowInput.md

```diff
@@ -0,0 +1,4 @@
+FlowInput
```

Comment:
The samples show usage of <Button> <Fields> under <FlowItem>. When and where will FlowInput be used?